src/renderer/lib/validate.ts

```diff
@@ -45,3 +45,21 @@ export const isValidConnectCodeStart = (codeStart: string) => {
 
   return true;
 };
+
+export const isValidDisplayName = (displayName: string): string | true => {
+  displayName = displayName.trim();
```

Comment:
I'm not a big fan of modifying function parameters like this tbh. I think the caller of the function should just pass in the trimmed version of the string.

```suggestion
```